techlibs/intel/synth_intel.cc

```diff
@@ -222,18 +220,10 @@ struct SynthIntelPass : public ScriptPass {
 		if (check_label("map_cells")) {
 			if (!noiopads)
 				run("iopadmap -bits -outpad $__outpad I:O -inpad $__inpad O:I", "(unless -noiopads)");
-			if (family_opt == "max10")
-				run("techmap -map +/intel/max10/cells_map.v");
-			else if (family_opt == "a10gx")
-				run("techmap -map +/intel/a10gx/cells_map.v");
-			else if (family_opt == "cyclonev")
-				run("techmap -map +/intel/cyclonev/cells_map.v");
-			else if (family_opt == "cyclone10")
-				run("techmap -map +/intel/cyclone10/cells_map.v");
-			else if (family_opt == "cycloneiv")
-				run("techmap -map +/intel/cycloneiv/cells_map.v");
-			else
-				run("techmap -map +/intel/cycloneive/cells_map.v");
+			string cmd = "techmap -map +/intel/FAMILY/cells_map.v";
+			cmd.replace(cmd.find("FAMILY"), 6, family_opt);
```

Comment:
It would be more Yosys-idiomatic to use `stringf` here, see things like https://github.com/YosysHQ/yosys/blob/master/techlibs/ice40/synth_ice40.cc#L340

Reply:
Fixed.